index.d.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference types="node"/>
+import {Duplex, DuplexOptions, Readable, Writable} from 'stream';
+
+export interface DuplexifyOptions extends DuplexOptions {
+  autoDestroy?: boolean;
+  end?: boolean;
+}
+
+export class Duplexify extends Duplex {
+  readonly destroyed: boolean;
+  setWritable(writable: Writable|false|null): void;
+  setReadable(readable: Readable|false|null): void;
```

Comment:
Did you purposefully omit `cork` and `uncork`?

Reply:
I did, I assumed these would get picked up from extending [`Duplex`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/v10/stream.d.ts#L221)

Reply:
Do you know what version it was added to node.js? Should be fine if it’s always been there, I didn’t know it was a part of node streams.

Reply:
Just checked, looks good. Been part of the API for a long time.